tests/test_init.py

```diff
@@ -17,542 +17,148 @@
-                            random_layer_normal,
-                            random_layers_normal,
-                            strong_ent_layer_uniform,
-                            strong_ent_layers_uniform,
-                            strong_ent_layer_normal,
-                            strong_ent_layers_normal)
+import pennylane as qml
+
+# Fix inputs to create fixture parameters
+n_wires = 3
+repeat = 2
+n_rots = 10
+n_if = n_wires * (n_wires - 1) / 2
+wrs = (repeat, n_wires)
+intf = (repeat, n_if)
+
+#######################################
+# Signatures
+
+base = {'n_wires': n_wires}
+rpt = {'n_layers': repeat, 'n_wires': n_wires}
+rpt_nrml = {'n_layers': repeat, 'n_wires': n_wires, 'mean': 0, 'std': 1}
+rpt_uni = {'n_layers': repeat, 'n_wires': n_wires, 'low': 0, 'high': 1}
+base_nrml = {'n_wires': n_wires, 'mean': 0, 'std': 1}
+base_uni = {'n_wires': n_wires, 'low': 0, 'high': 1}
+rnd_rpt_nrml = {'n_layers': repeat, 'n_wires': n_wires, 'n_rots': n_rots, 'mean': 0, 'std': 1}
+rnd_rpt_uni = {'n_layers': repeat, 'n_wires': n_wires, 'n_rots': n_rots, 'low': 0, 'high': 1}
+
+#######################################
+# Functions and their signatures
+
+# Functions returning a single parameter array
+INIT = [(qml.init.random_layers_normal, rnd_rpt_nrml),
+        (qml.init.strong_ent_layers_normal, rpt_nrml),
+        (qml.init.cvqnn_layers_theta_normal, rpt_nrml),
+        (qml.init.cvqnn_layers_phi_normal, rpt_nrml),
+        (qml.init.cvqnn_layers_varphi_normal, rpt_nrml),
+        (qml.init.cvqnn_layers_r_normal, rpt_nrml),
+        (qml.init.cvqnn_layers_phi_r_normal, rpt_nrml),
+        (qml.init.cvqnn_layers_a_normal, rpt_nrml),
+        (qml.init.cvqnn_layers_phi_a_normal, rpt_nrml),
+        (qml.init.cvqnn_layers_kappa_normal, rpt_nrml),
+        (qml.init.interferometer_theta_normal, base_nrml),
+        (qml.init.interferometer_phi_normal, base_nrml),
+        (qml.init.interferometer_varphi_normal, base_nrml),
+        (qml.init.random_layers_uniform, rnd_rpt_uni),
+        (qml.init.strong_ent_layers_uniform, rpt_uni),
+        (qml.init.cvqnn_layers_theta_uniform, rpt_uni),
+        (qml.init.cvqnn_layers_phi_uniform, rpt_uni),
+        (qml.init.cvqnn_layers_varphi_uniform, rpt_uni),
+        (qml.init.cvqnn_layers_r_uniform, rpt_uni),
+        (qml.init.cvqnn_layers_phi_r_uniform, rpt_uni),
+        (qml.init.cvqnn_layers_a_uniform, rpt_uni),
+        (qml.init.cvqnn_layers_phi_a_uniform, rpt_uni),
+        (qml.init.cvqnn_layers_kappa_uniform, rpt_uni),
+        (qml.init.interferometer_theta_uniform, base_uni),
+        (qml.init.interferometer_phi_uniform, base_uni),
+        (qml.init.interferometer_varphi_uniform, base_uni)
+        ]
+# Functions returning a list of parameter arrays
+ALL = [(qml.init.cvqnn_layers_all, rpt),
+       (qml.init.interferometer_all, base)]
+
+###########
+# Shapes
+
+SHP = [(repeat, n_rots), (repeat, n_wires, 3), intf, intf, wrs, wrs,
+       wrs, wrs, wrs, wrs, (n_if,), (n_if,), (n_wires,),
```

Comment:
this one was hard to parse 😄

Reply:
Any way we can make this part clearer to people who might read and/or modify these tests?

Reply:
One suggestion: just using full words (i.e., `wires` instead of `wrs`) might make the cognitive load slightly easier